demo/universal/native/lib/ServerFunctions.re

```diff
@@ -258,6 +288,66 @@ module Notes = {
 };
 
 module Samples = {
+  // Lets say this is the server action declared by the end-user
+  /**
+    [@react.server.action]
+    let formDataHandler = formData => {
+      let (_, name) = Hashtbl.find(formData, "name") |> List.hd;
+      let (_, lastName) = Hashtbl.find(formData, "lastName") |> List.hd;
+      let (_, age) = Hashtbl.find(formData, "age") |> List.hd;
+
+      Dream.log("Hello %s %s, you are %s years old", name, lastName, age);
+
+      Lwt.return(
+        (`String("Hello from server with form data action")),
+      );
+    };
+  */
+  // It's going to be on top to this that we are going to generate the codes bellow
+  let formDataId = "id/samples/form-data";
+
+  [@platform native]
+  let formDataHandler = formData => {
+    // For now, we are handling it by calling the value from Hashtbl
+    // We already have an issue to create FormData at Js.
+    let (_, name) = Hashtbl.find(formData, "name") |> List.hd;
+    let (_, lastName) = Hashtbl.find(formData, "lastName") |> List.hd;
+    let (_, age) = Hashtbl.find(formData, "age") |> List.hd;
+
+    Dream.log("Hello %s %s, you are %s years old", name, lastName, age);
+
+    Lwt.return("Hello from server with form data action");
+  };
+
+  // This is the router  handler that will handle parsing the args and calling the handler the user declared
+  // This code will be generated by the ppx automatically
+  // As the user didn't declare a request on the action, we don't need to pass it to the handler
+  [@platform native]
+  let formDataRouteHandler = formData => {
+    formDataHandler(formData) |> Lwt.map(response => `String(response));
+  };
+
+  let formData =
+    switch%platform () {
+    | Server => {
+        Runtime.React.id: Some(formDataId),
+        call: ((. formData) => formDataHandler(formData)),
+      }
+    | Client => {
+        Runtime.React.id: Some(formDataId),
+        call: (
+          (. formData) => {
+            let action =
+              ReactServerDOMWebpack.createServerReference(
```

Comment:
Same, why webpack?

Reply:
Ops, it was present on the previous branch and I didn't pay enough attention on it. Fixing